WordPress/Classes/Services/ApplicationPasswordService.swift

```diff
@@ -17,8 +17,8 @@ import WordPressAPI
 }
 
 extension ApplicationPasswordService: ApplicationTokenListDataProvider {
-    func loadApplicationTokens() async throws -> [ApplicationTokenItem] {
-        try await fetchTokens(forUserId: Int32(currentUserId))
+    func loadApplicationTokens(userId: Int32) async throws -> [ApplicationTokenItem] {
```

Comment:
(out of the scope) `Int32` is probably OK, but I'd model any IDs using `String` because they are more versatile.